projects/release-notes/docs/uitdatabank/2023.md

```diff
@@ -5,7 +5,7 @@
 ✨ **New features**
 
 * User interface: new entry form for creating organizers. This new entry form includes new features like the possibility to add a description and an image to your organizer, a completeness score for every item and many UX improvements.
-* User interface: new design. A simpel but more modern look in combination with clear icons provide a better user experience.
+* User interface: new design. A simple but more modern look in combination with clear icons provide a better user experience.
```

Comment:
The subject is third-person singular (A look, the rest are just adjectives) so I think it should be `provides` instead of `provide`.